signer/local/local_test.go

```diff
@@ -489,7 +489,12 @@ func TestOverrideSubject(t *testing.T) {
 
 	s := newCustomSigner(t, testECDSACaFile, testECDSACaKeyFile)
 
-	certPEM, err := s.Sign(signer.SignRequest{Hostname: "localhost", Request: string(csrPEM), Subject: req})
+	certPEM, err := s.Sign(signer.SignRequest{
+		Hosts:   []string{"127.0.0.1", "localhost"},
+		Request: string(csrPEM),
+		Subject: req,
+	})
+
```

Comment:
How should I parse certPEM? I tried using x509.ParseCertificate, but when I do I get

asn1: structure error: tags don't match (16 vs {class:0 tag:13 length:45 isCompound:true}) {optional:false explicit:false application:false defaultValue:<nil> tag:<nil> stringType:0 set:false omitEmpty:false} certificate @2

Is there some CFSSL function I should be using?